### DISCUSSION TITLE
Welcome to AR_Garage Discussions!

### DISCUSSION THREAD

**Discussion by remyar** [remyar/AR_Garage#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oguz69** [comment]
Bonjour j’ai téléchargé votre logiciel dont je vous félicite pour votre travail. Ce logiciel est bien pensé. Je suis auto entrepreneur. Lorsque j’indique le km du véhicule en impression le km n’apparaît pas. Pouvez-vous m’aider .de plus pour les paiements en espèces j’ai pas le choix. Faut il être connecté en mode admin? Mais ça demande un code . Pouvez-vous me donner ce code. Cordialement

**Reply by remyar** [reply]
normalement j'ai corriger quelques trucs et je vous ai repondu par mail mais je ne sais pas si vous avez eu la réponse.

**Comment by nl337** [comment]
Bonjour,
Je tenais à vous féliciter pour votre excellent travail sur le projet AR_Garage. C'est un outil très utile pour la gestion des devis et factures pour les garages en micro-entreprise.
J'aurais une question concernant l'accès au mode admin. Pourriez-vous me dire comment obtenir le mot de passe nécessaire pour accéder a la partie admin ? 
Merci d'avance pour votre aide et encore bravo pour votre projet !

**Reply by remyar** [reply]
Bonjour, la version Logiciel / Client lourd est laisser de coté pour mettre a profit la version Online dispo ici

https://ae-garage.fr

cette dernière embarque beaucoup de nouveauté. vous pouvez normalement exporter la base de donnée du logiciel PC pour l'importer dans le client Online.  

Cordialement

**Comment by nl337** [comment]
Bonjour,
Merci pour votre réponse concernant la version en ligne de l'application.
Je suis particulièrement intéressé par la personnalisation de la partie
véhicule du logiciel. Pourriez-vous me donner plus de détails sur les
options disponibles pour adapter cette section à mes besoins spécifiques ?
Je vous remercie par avance pour votre aide.

**Reply by remyar** [reply]
Pouvez vous m'envoyer un exemple de ce que vous souhaiteriez.

merci

**Comment by nl337** [comment]
Pour le véhicules j'aimerais juste avoir les informations tel que le
Chassis, la marque, la plaque, et le kilométrage aussi sur la facture
inséré la remise.

**Reply by remyar** [reply]
du coup, la plupart de ces informations sont présent sur la version online
